yarn.lock

```diff
@@ -2187,11 +2187,16 @@
   resolved "https://registry.yarnpkg.com/@types/ms/-/ms-0.7.31.tgz#31b7ca6407128a3d2bbc27fe2d21b345397f6197"
   integrity sha512-iiUgKzV9AuaEkZqkOLDIvlQiL6ltuZd9tGcW3gwpnX8JbuiuhFlEGmmFXEXkN50Cvq7Os88IY2v0dkDqXYWVgA==
 
-"@types/node@*", "@types/node@>= 8", "@types/node@^12.0.12", "@types/node@^14.14.6", "@types/node@^14.17.15":
+"@types/node@*", "@types/node@>= 8", "@types/node@^14.17.15":
```

Comment:
Are all these changes to `yarn.lock` expected? Many don't seem connected to the office-ui-fabric-react update.

Reply:
Yes. When I bumped the office-ui-fabric-react version, it appears that it updated also updated the yarn.lock entries for packages that are dependencies. I believe this happened in the "linking dependencies" and "build fresh packages" steps; that resulted in the reformatting (e.g. moving @types/node@^12.0.12" from line 2190 and creating a separate entry below on 2195.